site/docs/messaging/campaign-management/csp/campaign-api.mdx

```diff
@@ -87,7 +87,7 @@ HTTP/1.1 429 Too Many Requests
 | Campaign                   | Mandatory | Description                                                    |
 |:---------------------------|:----------|:---------------------------------------------------------------|
 | `BrandId`                  | Yes       |  Alphanumeric identifier of the brand associated with this campaign  |
-| `Vertical`                 | Yes       | 	Business/industry segment of this campaign. Must be of defined valid types |
+| `Vertical`                 | No        | 	Business/industry segment of this campaign. Must be of defined valid types or left blank/set to null |
```

Comment:
shouldn't this be in the api docs?

Reply:
Campaigns don't have API docs when it comes to CSP. It's only this guide

Reply:
We need to add these endpoints to the API docs.  Can you create a Jira for it?

Reply:
Created https://bandwidth-jira.atlassian.net/browse/MV-7841